schemata/cta.yaml

```diff
@@ -10,7 +10,11 @@ properties:
   id:
     "$ref": "base.json#/definitions/uuid"
   type:
-    "$ref": "contentblock.json#/definitions/type"
+    description: Block type
+    example: "text"
+    type: string
```

Comment:
@automata before this PR, any block would be valid as long as it passed the rules of _any_ block type. So `image` would be valid if it can pass `text` validations. Using stricter enum causes them to have to validate against the schema of the correct block type(s).